docker/data/add-data.sh

```diff
@@ -0,0 +1,216 @@
+#!/bin/bash
+
+FQDN="docker"
+DATADIR="/data"
+PSQL="psql -U bety -h postgres -d bety -q -t -c"
```

Comment:
Should some (or all) of these be configurable (with the current values as defaults)? Or is there a Very Good Reason for these to be hard-coded as they are? I'm especially interested in `FQDN` -- I'm still not 100% clear on where we always use `"docker"` vs. where it will be machine-specific.

Reply:
The FQDN should be  a variable. This is the name of the machine used in the machines table and should be  FQDN. using the name of the docker container will make it random, hence we use the specified name. See also #2128 and #2186.

Reply:
Added to documentation quickstart as well.